pkg/jx/cmd/import.go

```diff
@@ -218,7 +218,7 @@ func (o *ImportOptions) Run() error {
 		config := authConfigSvc.Config()
 		var server *auth.AuthServer
 		if o.RepoURL != "" {
-			gitInfo, err := gits.ParseGitURL(o.RepoURL, o.GitProvider.Kind() == "bitbucketserver")
+			gitInfo, err := gits.ParseGitURL(o.RepoURL, false)
```

Comment:
Hi @ccojocar ,

We would need to send `true` if the gitprovider is bitbucketserver.

How about we try this instead?

```
var hasSCM bool
if o.GitProvider != nil && o.GitProvider.Kind() == "bitbucketserver" {
  hasSCM = true
} else {
  hasSCM = false
}

if o.RepoURL != "" {
  gitInfo, err := gits.ParseGitURL(o.RepoURL, hasSCM)
  ...
```

Reply:
Sorry I broke the functionality with my PR :(